packages/cli/src/constructs/check-group-v1.ts

```diff
@@ -0,0 +1,536 @@
+   *
+   * Currently only the following settings have an effect:
+   * - {@link BrowserCheckConfig.frequency}
+   */
+  browserChecks?: BrowserCheckConfig,
+
+  /**
+   * This optional setting can be used to provide CLI-level defaults for
+   * all multi-step checks in the group.
+   *
+   * Note that any settings defined here are stored at the check level and are
+   * not persisted in the group.
+   *
+   * Currently only the following settings have an effect:
+   * - {@link MultiStepCheckConfig.frequency}
+   */
+  multiStepChecks?: MultiStepCheckConfig,
+
+  /**
+   * If set, all checks in the group will use the group's alert escalation
+   * policy.
+   *
+   * If not set, the implicit default policy is:
+   * `AlertEscalationBuilder.runBasedEscalation(1)`.
+   */
+  alertEscalationPolicy?: AlertEscalation,
+
+  /**
+   * A valid piece of Node.js code to run in the setup phase of an API check
+   * in this group.
+   *
+   * @deprecated Use the "ApiCheck.setupScript" property instead and use
+   * common JS/TS code composition to add group specific setup routines.
+   */
+  localSetupScript?: string
+
+  /**
+   * A valid piece of Node.js code to run in the teardown phase of an API
+   * check in this group.
+   *
+   * @deprecated use the "ApiCheck.tearDownScript" property instead and use
+   * common JS/TS code composition to add group specific teardown routines.
+   */
+  localTearDownScript?: string
+
+  /**
+   * If set, the values provided here are merged with the values defined
+   * in individual API checks.
+   */
+  apiCheckDefaults?: ApiCheckDefaultConfig
+
+  /**
+   * Sets a retry policy for the group. Use {@link RetryStrategyBuilder} to
+   * create a retry policy.
+   *
+   * If set, all checks in the group use the group's retry strategy.
+   *
+   * If not set, retries are disabled for all checks in the group.
+   */
+  retryStrategy?: RetryStrategy
+
+  /**
+   * Determines whether the checks in the group should run on all selected
+   * locations in parallel or round-robin.
+   *
+   * When `true`, all checks in the group run in parallel regardless of their
+   * individual setting.
+   *
+   * When `false`, all checks in the group run in round-robin regardless of
+   * their individual setting.
+   *
+   * If not set, the default is `false`.
+   *
+   * See https://www.checklyhq.com/docs/monitoring/global-locations/ to learn
+   * more about scheduling strategies.
+   */
+  runParallel?: boolean
+}
+
+/**
+ * Creates a Check Group (v1).
+ *
+ * We strongly recommend upgrading to {@link CheckGroupV2} instead.
+ *
+ * The original CheckGroup v1 comes with implicit defaults that are not
+ * intuitive and make it impossible to keep check-level behavior for
+ * certain properties.
+ *
+ * The following properties will always use the group's values if set, or
+ * implicit defaults if not set:
+ *
+ *   - {@link CheckGroupV1Props.alertEscalationPolicy}
+ *   - {@link CheckGroupV1Props.retryStrategy}
```

Comment:
are we missing "locations" here?

Reply:
No because the backend has already been changed in a way that makes it apply to all groups. There is no difference in behavior between `CheckGroup` and `CheckGroupV2`.